### PR TITLE
WT-9348 Merge import_compatibility_test.sh into compatibility_test_for_releases.sh

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

File: test/evergreen/compatibility_test_for_releases.sh [hunk 1]
```diff
@@ -676,6 +690,134 @@ top="test-compatibility-run"
 rm -rf "$top" && mkdir "$top"
 cd "$top"
 
+#############################################################
```
Comment (lukech) [inline test/evergreen/compatibility_test_for_releases.sh:693]: Suggest moving these function definitions up (around line 530).

File: test/evergreen/compatibility_test_for_releases.sh [hunk 2]
```diff
@@ -676,6 +690,134 @@ top="test-compatibility-run"
...
+# import_file:
+#       arg1: dest branch
+#       arg2: source branch
+#       arg3: file
+#############################################################
+import_file()
+{
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    echo "Importing file \"$3\" from \"$1\" to \"$2\""
```
Comment (lukech) [inline test/evergreen/compatibility_test_for_releases.sh:725]: Should it be "Importing file \"$3\" from \"$2\" to \"$1\"" instead? It feels more intuitive to have arg1 being the source and arg2 the destination :)

**Review by ruby-oujo — APPROVED** [review]
LGTM! :)

File: test/evergreen/compatibility_test_for_releases.sh (hunk 2 above)
Comment (mm-ng) [reply test/evergreen/compatibility_test_for_releases.sh:725]: Good catch, thanks for that. Yeah I agree with your sentiment about that, I'll make changes to other functions if needed as well.

**Review by lukech — APPROVED** [review]
LGTM, thanks.

**Pull request merged**